libraries/RC_Channel/RC_Channel.h

```diff
@@ -352,6 +359,9 @@ class RC_Channels {
         return _override_timeout.get() * 1e3f;
     }
 
+    // primary input control channels
+    RC_Channel *channel_throttle;
```

Comment:
I understand that you were trying to reduce code duplication, but Tracker for example doesn't seem to have this.

Reply:
Yeah.  I think I'll remove this assumption.